source/languages/en/riak/dev/advanced/bucket-types.md

```diff
@@ -333,6 +338,18 @@ var rslt1 = client.Get(id1);
 var rslt2 = client.Get(id2);
 ```
 
+```javascript
+client.fetchValue({
+    bucketType: 'type1', bucket: 'my_bucket', key: 'my_key'
+}, function (err, rslt) {
```

Comment:
Is this the intended indentation level? Looks... well, to say it looks kinda weird doesn't really mean anything when you're talking about JS. It's not what I expected. That might say more about my expectations than the code.